client/src/consts/Milvus.tsx

```diff
@@ -19,34 +19,34 @@ export const METRIC_TYPES = [
   },
   {
     value: METRIC_TYPES_VALUES.SUBSTRUCTURE,
-    label: 'Substructure',
+    label: 'SUBSTRUCTURE',
```

Comment:
Why we need to uppercase metric options label?

Reply:
Milvus need all uppercase value right now.so may be it's better to make label uppercase too.